xwiki-platform-distribution/xwiki-platform-distribution-war-legacydependencies/pom.xml

```diff
@@ -319,5 +319,17 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>org.xwiki.platform</groupId>
+      <artifactId>xwiki-platform-legacy-extension-handler-xar</artifactId>
+      <version>${project.version}</version>
+      <!-- Exclude transitive dependencies which are conflicting with legacy dependencies -->
```

Comment:
@tmortagne I spotted that I should exclude oldcore but I didn't perform a deep analysis here: is the build supposed to fail in case I missed some? Or do I need manually to check the deps?

Reply:
If oldcore is not excluded, we end up with both legacy-oldcore and oldcore in the WAR which should cause the duplicate-finder-maven-plugin plugin to fail the build (because we end up with JARs which contain classes and resources with the same name).

Reply:
ok, so since the build with quality profile is working there it should be enough then I guess.